heavyai/_pandas_loaders.py

```diff
@@ -246,6 +253,20 @@ def _serialize_arrow_payload(data, table_metadata, preserve_index=True):
         else:
             data_ = data
 
+        # convert geo columns to WKT representation
+        for col in table_metadata:
+            if col.type in GEO_TYPE_NAMES:
+                try:
+                    fn = functools.partial(shapely.to_wkt,
+                                           rounding_precision=-1,
+                                           trim=False,
+                                           output_dimension=2)
+                    data_[col.name] = data_[col.name].apply(fn)
+                except TypeError:
+                    msg = (f"Column '{col.name}' is not a geometry column. "
+                           "Please check your input data.")
```

Comment:
This sounds like a very vague suggestion for end-users to process considering that they cannot do much when a third-party library produces arrow buffers that pandas cannot load.

If `col.type` is in `GEO_TYPE_NAMES`, why `to_wkt` would fail anyway? And if it does, perhaps this is a software bug or limitation rather than users data issue that the exception message suggests.

Reply:
That's true. It is unlikely that users will face this exception.